README.md

```diff
@@ -7,90 +7,278 @@
 - Explain adding and committing changes
 - Explain pushing branches
 - Identify local and remote branches
-- Explain pulling branches
+- Explain getting updates from remote
 - Explain merging branches
 - Explain deleting branches
```

Comment:
This might be a discussion topic for the team, but I've started to prefer writing learning goals based on the actions we expect the students to take. For example, instead of "Explain merging branches," we use "Merge branches." Or instead of "Use git log to review changes", we use "Review changes with git log." That way we're emphasizing what the students need to focus on doing.